docs/pages/server-access/getting-started.mdx

```diff
@@ -76,42 +89,63 @@ the rights and access to read.
 
 <ScopedBlock scope={["oss", "enterprise"]}>
 
-Start the Node. Change `tele.example.com` to the address of your Teleport Proxy
+Configure and start the Node. Change `tele.example.com` to the address of your Teleport Proxy
```

Comment:
```suggestion
Configure and start the SSH Service instance. Change `tele.example.com` to the address of your Teleport Proxy
```

To be more explicit—we took "Node" out of the Core Concepts page (https://goteleport.com/docs/core-concepts/).

Reply:
addressed